test/e2e/storage/volume_provisioning.go

```diff
@@ -1247,7 +1169,12 @@ func deleteProvisionedVolumesAndDisks(c clientset.Interface, pvs []*v1.Persisten
 func getRandomCloudZone(c clientset.Interface) string {
 	zones, err := framework.GetClusterZones(c)
 	Expect(err).ToNot(HaveOccurred())
+
 	// return "" in case that no node has zone label
-	zone, _ := zones.PopAny()
-	return zone
+	if zones.Len() == 0 {
+		return ""
```

Comment:
this seems like it should be an error case, I don't think any of the callers check for this